### ISSUE TITLE
PyCharm not finding classes

### ISSUE THREAD

**Issue by GitRon** [theatlantic/django-nested-admin#198]
I hope I'm not reacting a duplicate issue, but couldn't find it in the first sweep.

![grafik](https://user-images.githubusercontent.com/3176075/112274858-4158b580-8c7f-11eb-82bc-8df3d558363c.png)

Pycharm is not detecting any classes thus it cannot help suggesting attributes etc. Plus I get loads of warnings which I try to avoid.

Any ideas on the topic?

Thanks!  
Ronny

**Comment by drsounds** [comment]
Hi,

I encounted the same problem. For me I found out you nowmore seems to have to import from nested_admin.nested
````python
    from nested_admin.nested import NestedGenericTabularInline
````

**Comment by GitRon** [comment]
Yes, this works. Still, could be easier to do or should be at least documented.

I went with this approach:

```
from nested_admin import nested

class MyNestedInline(nested.NestedGenericTabularInline):
    pass
```
